modules/solid_mechanics/src/auxkernels/MaterialTensorCalculator.C

```diff
@@ -88,54 +89,58 @@ MaterialTensorCalculator::getTensorQuantity(const SymmTensor & tensor,
       break;
 
     case 3:
-      value = MaterialTensorCalculatorTools::hydrostatic(tensor);
+      value = MaterialTensorCalculatorTools::effectiveStrain(tensor);
       break;
 
     case 4:
-      value = MaterialTensorCalculatorTools::directionValueTensor(tensor, _direction);
+      value = MaterialTensorCalculatorTools::hydrostatic(tensor);
       break;
 
     case 5:
-      value = MaterialTensorCalculatorTools::hoopStress(tensor, _p1, _p2, curr_point, direction);
+      value = MaterialTensorCalculatorTools::directionValueTensor(tensor, _direction);
       break;
 
     case 6:
-      value = MaterialTensorCalculatorTools::radialStress(tensor, _p1, _p2, curr_point, direction);
+      value = MaterialTensorCalculatorTools::hoopStress(tensor, _p1, _p2, curr_point, direction);
       break;
 
     case 7:
-      value = MaterialTensorCalculatorTools::axialStress(tensor, _p1, _p2, direction);
+      value = MaterialTensorCalculatorTools::radialStress(tensor, _p1, _p2, curr_point, direction);
       break;
 
     case 8:
-      value = MaterialTensorCalculatorTools::maxPrinciple(tensor, direction);
+      value = MaterialTensorCalculatorTools::axialStress(tensor, _p1, _p2, direction);
       break;
 
     case 9:
-      value = MaterialTensorCalculatorTools::midPrinciple(tensor, direction);
+      value = MaterialTensorCalculatorTools::maxPrinciple(tensor, direction);
       break;
 
     case 10:
-      value = MaterialTensorCalculatorTools::minPrinciple(tensor, direction);
+      value = MaterialTensorCalculatorTools::midPrinciple(tensor, direction);
       break;
 
     case 11:
-      value = MaterialTensorCalculatorTools::firstInvariant(tensor);
+      value = MaterialTensorCalculatorTools::minPrinciple(tensor, direction);
```

Comment:
While you're in here, it would be good to fix the spelling of these.  I'd change the name of the function to `minPrincipal()`, etc., and then add functions with names like `minPrinciple()` that just call through to the correctly-spelled versions for backward compability while we go fix everything that calls these.  `tensor_mechanics` has the same issue.